pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.amazon.deequ</groupId>
     <artifactId>deequ</artifactId>
-    <version>2.0.3-spark-3.3</version>
+    <version>2.0.4-spark-3.4</version>
```

Comment:
I have incremented the build version. Let me know if this should be separated into different PR.